lib/src/components/SimpleComponents.tsx

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import withClass from '../util/withClass';
+
+// Simple convenience components that are just a particular tag with a particular classname, with the ability
+// to add all attrs that you would normally be able to add to that tag (including more classnames)
+export const NxP = withClass('nx-p', 'p');
+export const NxCode = withClass('nx-code', 'code');
+export const NxBlockquote = withClass('nx-blockquote', 'blockquote');
+export const NxPre = withClass('nx-pre', 'pre');
+export const NxFooter = withClass('nx-footer', 'footer');
+export const NxButtonBar = withClass('nx-btn-bar', 'div');
+export const NxCounter = withClass('nx-counter', 'span');
+export const NxFormSelect = withClass('nx-form-select', 'select');
+export const NxThreatNumber = withClass('nx-threat-number', 'span');
+export const NxPageMain = withClass('nx-page-main', 'main');
+export const NxPageSidebar = withClass('nx-page-sidebar', 'aside');
+
+export const NxH1 = withClass('nx-h1', 'h1');
+export const NxH2 = withClass('nx-h2', 'h2');
+export const NxH3 = withClass('nx-h3', 'h3');
+export const NxH4 = withClass('nx-h4', 'h4');
+
+export const NxTile = Object.assign(withClass('nx-tile', 'section'), {
+  Header: withClass('nx-tile-header', 'header'),
```

Comment:
If we're attaching these as child properties on the parent component, should we consider doing something similar for other components like the `NxTable` family of components? Or should these become NxTileHeader etc to match the established pattern in the RSC library?

Reply:
> If we're attaching these as child properties on the parent component, should we consider doing something similar for other components like the NxTable family of components? Or should these become NxTileHeader etc to match the established pattern in the RSC library?

I've been drifting more towards the style I have here.  `NxAccordion` was done the same way.  We could put in a ticket to set up `NxTable` the same way and deprecate the old top-level names for its children

Reply:
Sounds like it will be a breaking change either way, but I think it makes sense to make the change for consistency. If it's a large burden for existing code to change I suppose we could provide an alias in the index file.

Reply:
> we could provide an alias in the index file.

Right that's what I had in mind with deprecating them.  I've added RSC-543 for this.